lib/srv/monitor.go

```diff
@@ -194,6 +194,7 @@ func (c *ConnectionMonitor) MonitorConn(ctx context.Context, authzCtx *authz.Con
 		Clock:                 c.cfg.Clock,
 		ServerID:              c.cfg.ServerID,
 		TeleportUser:          identity.Username,
+		UserOriginClusterName: identity.TeleportCluster,
```

Comment:
Is this correct? Shouldn't we go through authzCtx.UnmappedIdentity?

Reply:
that's correct. And it looks like we change the cluster in the mapped identity. Since we use for audit events Identity, I added an extra field to tlsca

Reply:
updated